subscription_gateway.go

```diff
@@ -53,3 +56,33 @@ func (g *SubscriptionGateway) Cancel(ctx context.Context, subId string) (*Subscr
 	}
 	return nil, &invalidResponseError{resp}
 }
+
+// RetryCharge retries to charge for a subscription. The amount has to
+// be > 0.
+func (g *SubscriptionGateway) RetryCharge(ctx context.Context, subId string, amount Decimal) error {
+	txInput := &struct {
+		XMLName        xml.Name
+		Amount         Decimal            `xml:"amount"`
+		Options        TransactionOptions `xml:"options"`
+		SubscriptionID string             `xml:"subscription-id"`
+		Type           string             `xml:"type"`
+	}{
+		XMLName: xml.Name{Local: "transaction"},
```

Comment:
I think you should be able to move the `"transaction"` into a tag on the `xml.Name` field like:

```
XMLName xml.Name `xml:"transaction"`
```